app/controllers/orders/index.js

```diff
@@ -25,6 +25,10 @@ export default Ember.Controller.extend(SearchMixin, {
     }
   },
 
+  hasSearchText: Ember.computed("searchText", function() {
```

Comment:
This property is also newly added. Where are you using it ?
Did you forget to commit the template file ?

Reply:
So the action is already present hbs file but the property and actions were not present in respective js file,so added that property and action for the same

Reply:
ohh i see. Thanks for clarifying.